src/commands/ipmi.rb

```diff
@@ -69,7 +88,7 @@ def ipmi_command(node, arguments:)
       end
 
       def command_argument
-        args[1]
+        Command.parse(args, options)
```

Comment:
Because we're not attempting to parse the command argument/option until we use `command_argument`, which happens after other parts of the `ipmi` command such as loading parts of the node's config, this can lead to additional possibly confusing output:

```
[root@controller(dev) metalware]# bin/metal ipmi node01
warning: The following returned nil: answer.is_vm
warning: The following returned nil: answer.libvirt_host
error: No command given. Use --help for more information.
```

Possibly we should parse the `command_argument` in a `setup` method, so that the command argument is either successfully parsed up-front or the command fails, rather than having additional warnings potentially printed before informing the user that their command line itself is invalid. `command_argument` could then be accessed via an `attr_reader`.